app.js

```diff
@@ -19,9 +18,20 @@ var app = express();
 app.set('views', path.join(__dirname, 'views'));
 app.set('view engine', 'ejs');
 
+// Authentication 
+app.use(session({
+  secret: 'secret',
+  resave: true,
+  saveUninitialized: false,
+  cookie: { maxAge: 60000, secure: false }
```

Comment:
maxAge in milliseconds, so this is JUST 1 minute. Suggest moving to 1 hour (360000)

Reply:
Ok noted